client/src/components/TabularViewComponent/TabularViewComponent.scss

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+@import "../../theme/theme.scss";
+
+.ms-DetailsList {
+    overflow-x: hidden;
+}
+
+.ms-Viewport  {
+    width: 1100px;
```

Comment:
Similar comment... do we need to fix the width?

Reply:
Change this into a 90% of the available Layer